cloud_service/getting-started-rhacs-cloud-ocp.adoc

```diff
@@ -27,39 +27,76 @@ The following sections provide an overview of installation steps and links to th
 [id="securing-rh-cloud-clusters"]
 === Securing {osp} clusters
 
-To secure {osp} clusters by using the Operator, perform the following steps:
+You can secure {osp} clusters by using the Operator, Helm charts, or the `roxctl` CLI.
+
+[id="overview-installing-cloud-secured-clusters-osp-operator"]
+==== Securing {osp} clusters by using the Operator
 
 . Verify that the clusters you want to secure meet the xref:../cloud_service/acscs-default-requirements.adoc#acscs-default-requirements[requirements].
 . In the {cloud-console}, xref:../cloud_service/installing_cloud_ocp/cloud-create-instance-ocp.adoc#cloud-create-instance-ocp[create an *ACS Instance*].
 . On each {osp} cluster you want to secure, xref:../cloud_service/installing_cloud_ocp/cloud-ocp-create-project.adoc#cloud-ocp-create-project[create a project named `stackrox`]. This project will contain the resources for {product-title-managed-short} secured clusters.
-. In the ACS Console, xref:../cloud_service/installing_cloud_ocp/init-bundle-cloud-ocp-generate.adoc#init-bundle-cloud-ocp-generate[create an init bundle]. The init bundle contains secrets that allow communication between {product-title-managed-short} secured clusters and the ACS Console.
-. On each {osp} cluster, xref:../cloud_service/installing_cloud_ocp/init-bundle-cloud-ocp-apply#init-bundle-cloud-ocp-apply[apply the init bundle] by using it to create resources.
+. Create the mechanism that the Central instance, also called Central, will use to set up communication with the secured clusters. You will either create an init bundle or a cluster registration secret (CRS). Do one of these actions:
```

Comment:
```suggestion
. Create the mechanism that the Central instance, also called Central, will use to set up communication with the secured clusters. You can either create an init bundle or a cluster registration secret (CRS). Do one of these actions:
```

Reply:
I personally prefer "perform" but I was told not to use it in a previous peer review (although [IBM Style guidance](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#perform) is kind of...not hardline on it).